token.go

```diff
@@ -178,3 +179,32 @@ func (m TokenModule) postTokenHandler(c *gin.Context) {
 
 	c.JSON(http.StatusCreated, token)
 }
+
+// postTokenHandler godoc
+// @summary Put token in database.
+// @description Put token in database. Provider in post object has to exists or should be empty.
+// @description Provider will has to be updated token ID during this operation.
```

Comment:
These "provider" notices here are no longer relevant.

```suggestion
// @description Creates a new token if a match is not found.
```